code/modules/mob/living/living.dm

```diff
@@ -1657,13 +1657,13 @@ Thanks.
 				returnToPool(G)
 	if(!item)
 		return FAILED_THROW	//Grab processing has a chance of returning null
+	if(isitem(item))
+		var/obj/item/I = item
+		if(I.cant_drop > 0)
+			to_chat(usr, "<span class='warning'>It's stuck to your hand!</span>")
+			return FAILED_THROW
 
-	var/obj/item/I = item
-	if(istype(I) && I.cant_drop > 0)
```

Comment:
Wouldn't the `istype(I)` prevent this `if()` from succeeding if `item` is not an item?
If you give `istype()` one argument, it checks whether the given variable is of the type as which it is cast.